test/powershell/Modules/Microsoft.PowerShell.Core/Enter-PSHostProcess.Tests.ps1

```diff
@@ -152,8 +167,8 @@ Describe "Enter-PSHostProcess tests" -Tag Feature {
 
                 $result | Should -Be $pwshId -Because $errorMsg
             } finally {
-                $rs.Dispose()
-                $ps.Dispose()
+                ${rs}?.Dispose()
+                ${ps}?.Dispose()
```

Comment:
@adityapatwardhan 😎

Reply:
Don't use experimental feature in tests :(
This will not work in a RC/GA release if this feature is not made permanent. Lets refrain from taking dependencies on experimental features in tests.

Reply:
Changed to normal null check.